libcobblersignatures/__init__.py

```diff
@@ -87,6 +87,8 @@ def importsignatures(self, import_type: ImportTypes, source: str):
             self._importsignaturesurl(source)
         elif import_type == ImportTypes.STRING:
             self.signaturesjson = source
+        elif import_type == ImportTypes.BUILD_IN:
+            self._importsignaturesfile("libcobblersignatures/data/v2/distro_signatures.json")
```

Comment:
This won't work. We will need to use importlib (and its backport) as described in the issue that I linked. The location of the file that you gave here is relative to the current directory a user is in. As such this will break if the user is not in the project root (which he won't be in if the package is installed via pip).